device/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:mysql://localhost:3306/webStore?createDatabaseIfNotExist=true&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/brainHouse?createDatabaseIfNotExist=true&serverTimezone=UTC
```

Comment:
don't replace property values with your own.

Reply:
It is wrong database name. so I need to change it